### DISCUSSION TITLE
Minting site and NFT has been marked as malicious

### DISCUSSION THREAD

**Discussion by BennjjiB** [phantom/docs#332] [Help & Support] [answered]
### Summary

Hello Phantom Team, we've been developing a Mint site to presale NFTs for a card game called Otium. 
Both the site and the NFTs have been marked as spam and malicious by the Phantom Wallet. 
We already tried contacting blowfish.xyz but they don't seem to be responding. 
Do you have any other suggestions, what steps we could take to get the site and NFTs whitelisted ?
Any suggestions are greatly appreciated!

Site link: 
- https://www.mint.playotium.com/  (Presale of NFTs)
- https://www.mint.playotium.com/raffle (Site to buy weekly raffle tickets)

NFT Details:
All the NFTs are part of a collection NFT:
https://solscan.io/token/AfgLivczGEy9em6SUXLTy47Teq2aUG9rDnePrPgNmKme 
Here is an example:
https://solscan.io/token/38no6Cmo3jx6jmds2eCkjzLCsLLsQFR6GtKAgotsVCKM#metadata 

Links to the projects channels:
X: https://twitter.com/0xTheNotwork
Website: https://thenotwork.cc/
Website: https://playotium.com/
YouTube: https://www.youtube.com/@Notworkers
Reddit: https://www.reddit.com/r/TheNotwork/
TeleGram: https://t.me/+6i1fEO0v491lYTdh
TikTok: https://www.tiktok.com/@otiumofficial?lang=en
Discord: https://discord.gg/5HzVF8KS8w
Chart: https://ape.pro/solana/GcdLTfPGhdsX6zVjmcLchwwECzYqATHgk64sKZuadHKF

### Example

https://www.mint.playotium.com/ 

### Steps to Reproduce

1. Connect a wallet 
2. Keep the Knights chest selected with amount equal to 1
3. Click the 'Mint for 0.125 $Sol' button 
4. A dialog from the phantom wallet should open looking like this:
<img width="1054" alt="Screenshot 2024-12-09 at 22 00 59" src="https://github.com/user-attachments/assets/56b94406-1bfd-43d5-9322-371625eeb2bf">

### Phantom Version

24.27.1

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@BennjjiB Please email into review@blowfish.xyz. They have not received a request regarding this dapp.

**Reply by BennjjiB** [reply]
@adamdelphantom thank you for checking in with them, we will try contacting blowfish again.

**Reply by BennjjiB** [reply]
Hi @adamdelphantom after contacting blowfish again and giving them everything we could (social media, website and so forth) even made the github for the sites open, they've responded with what is basically sorry but no. With the reason, that we don't have anybody to vouch for us and are to new to the solana ecosystem. That's quite a bummer. Is there any other way to get the sites approved?
